example/fusionauth_dart_client_auth_example.dart

```diff
@@ -0,0 +1,41 @@
+import 'package:fusionauth_dart_client/fusionauth_dart_client.dart'
+    as fusionauth;
+
+void main() async {
+  Future<fusionauth.ClientResponse<fusionauth.LoginResponse, fusionauth.Errors>>
+      processAuth(fusionauth.LoginRequest loginRequest) async {
+    var client = fusionauth.FusionAuthClient(
+        'bf69486b-4733-4470-a592-f1bfce7af580', //API Key
+        'https://local.fusionauth.io',
+        null);
+
+    var authResult = await client.login(loginRequest);
+
+    return authResult;
+  }
+
+  var loginRequest = fusionauth.LoginRequest(
+      loginId: 'user@domain.com', password: 'Password!');
+
+  loginRequest.applicationId = '9e0f97c2-5563-4b81-83a8-1db6e6457906';
+  loginRequest.noJWT = false;
+  loginRequest.metaData = fusionauth.MetaData(
+      device: fusionauth.DeviceInfo(
+          name: 'IPhone XR', //DeviceName
+          type: fusionauth.DeviceType.MOBILE));
+
+  var result = await processAuth(loginRequest);
+
+  if (result.statusCode == 200) {
```

Comment:
Hmmm. We usually just check for anything between 200 and 299 for success, because there are various 2xx status codes: https://fusionauth.io/docs/v1/tech/apis/login#response

Can you change that?